tinyphysics.py

```diff
@@ -69,7 +69,7 @@ def softmax(self, x, axis=-1):
     e_x = np.exp(x - np.max(x, axis=axis, keepdims=True))
     return e_x / np.sum(e_x, axis=axis, keepdims=True)
 
-  def predict(self, input_data: dict, temperature=1.) -> dict:
+  def predict(self, input_data: dict, temperature=1.) -> int:
```

Comment:
I guess `np.ndarray` is more appropriate?

Reply:
np.random.choice returns an int in this case.

Reply:
You're right!